ogre2/src/Ogre2ThermalCamera.cc

```diff
@@ -1130,6 +1130,19 @@ void Ogre2ThermalCamera::PreRender()
 {
   if (!this->dataPtr->ogreThermalTexture)
     this->CreateThermalTexture();
+
+  // todo(iche03) Override BaseCamera::SetProjectionMatrix() function
```

Comment:
is this comment releted with the next version ? because it's breaking ABI, isn't it ? can you include in which version we should do this ?

Reply:
updated comments to mention the todo version. [868cc4a](https://github.com/gazebosim/gz-rendering/pull/1002/commits/868cc4ab7560fc56fc3df75ce1322bcc859fe075)